shell/src/main/java/alluxio/cli/fs/command/CatCommand.java

```diff
@@ -54,7 +55,7 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
     if (status.isFolder()) {
       throw new FileDoesNotExistException(ExceptionMessage.PATH_MUST_BE_FILE.getMessage(path));
     }
-    OpenFilePOptions options = OpenFilePOptions.getDefaultInstance();
+    OpenFilePOptions options = FileSystemOptions.openFileDefaults(mFileSystem.getConf());
```

Comment:
This is passed to a FileSystem. Merging should be done there.

Reply:
fixed